jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/wait/LogMatchCallback.java

```diff
@@ -24,7 +24,7 @@ class LogMatchCallback implements LogCallback {
     private final KitLogger logger;
     private final LogWaitCheckerCallback callback;
     private final Pattern pattern;
-    private StringBuilder logBuffer;
```

Comment:
Hi, the issue is only related to the LogRequestor.java file, please revert changes to this file.

Reply:
@manusa can you check now?